AK/StringUtils.cpp

```diff
@@ -346,6 +346,19 @@ bool is_whitespace(StringView str)
     return all_of(str, is_ascii_space);
 }
 
+StringView trim_at_first_null(StringView str)
```

Comment:
This is just
`view.substring_view(0, view.find('\0').value_or(view.length());`

I'm not sure we need a helper for this?

Either way, this isn't really a "trim" operation, it's a `first substring` operation.